### PR TITLE
Added commands

### PR DESCRIPTION
**Pull request by madams51703**
Added alter broker_priority thru alter cyrptographic provider, Minus Availability Groups.  

Also replaced tabs I introduced with spaces.

I hope these rules look a little better.

### CODE REVIEW

File: tsql/TSqlLexer.g4 [hunk 1]
```diff
@@ -366,7 +373,9 @@ DOCUMENT:                              'DOCUMENT';
 DYNAMIC:                               'DYNAMIC';
 EMERGENCY:                             'EMERGENCY';
 EMPTY:                                 'EMPTY';
+ENABLE:                                'ENABLE';
 ENABLE_BROKER:                         'ENABLE_BROKER';
+ENCRYPTED_VALUE:                        'ENCRYPTED_VALUE';
```
Comment (KvanTTT) [inline tsql/TSqlLexer.g4]: Unaligned literal.

File: tsql/TSqlParser.g4 [hunk 2]
```diff
@@ -472,9 +477,44 @@ class_type_for_parallel_dw
     | OBJECT
     ;
 
+//https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-broker-priority-transact-sql
```
Comment (KvanTTT) [inline tsql/TSqlParser.g4]: Please add at least one white space after comment slashes for better readability.

**Comment by KvanTTT** [comment]
Overall good job!

**Comment by madams51703** [comment]
Thanks.  Making those changes.

**Pull request merged**